docs/source/development.md

```diff
@@ -89,5 +89,6 @@ The C++ libraries in `obelisk/cpp` are built with CMake.
 
 All `obelisk` C++ libraries are placed in `obelisk/`. Each library should have its own folder. Within that folder there should be an `include` folder, a `CMakeLists.txt`, and the source files (i.e., not the header files - those go in `include`).
 
+(testing)=
```

Comment:
i removed this before, what does this do?

Reply:
This lets us have an internal page reference. So up above I reference this section, which can only happen with this tag.